web/src/popover_menus_data.js

```diff
@@ -118,3 +125,23 @@ export function get_actions_popover_content_context(message_id) {
         should_display_quote_and_reply,
     };
 }
+
+export function get_topic_popover_content_context({$stream_li, topic_name, url}) {
+    const stream_id = stream_popover.elem_to_stream_id($stream_li);
```

Comment:
I thought I'd asked that `stream_popover` not be imported in this module, since it'll create an import cycle. I'll see if I can just undo this before merging.

Reply:
Sure, thanks a lot for that, do let me know if you want me to make any more changes :)

Reply:
Ahh, indeed https://github.com/zulip/zulip/pull/24783#discussion_r1157654950